src/Tickets/Commerce/Settings.php

```diff
@@ -327,8 +333,35 @@ public function get_settings() {
 				'validation_type'     => 'textarea',
 			],
 		];
-
-		$settings = array_merge( $gateways_manager->get_gateway_settings(), $settings );
+		
+		// Add featured settings to top of other settings.
+		$admin_views = tribe( Tribe__Tickets__Admin__Views::class );
+		$featured_settings = [
+			'tc_featured_settings' => [
+				'type' => 'html',
+				'html' => tribe( Featured_Settings::class )->get_html([
+					'title'            => __( 'Payment Gateways', 'event-tickets' ),
+					'description'      => __( 
+						'Set up a payment gateway to get started with Tickets Commerce. Enable multiple ' .
+						'gateways for providing users additional options for users when purchasing tickets.', 
+						'event-tickets' 
+					),
+					'content_template' => $admin_views->template( 'settings/tickets-commerce/gateways', [], false ),
+					'links'            => [
+						[
+							'slug'         => 'help-1',
+							'priority'     => 10,
+							'link'         => esc_url( '#' ),
```

Comment:
Maybe we can leave a `@todo` here to be sure that we take care of this one later on?